### PR TITLE
Correct "Tesing" in "test_sims.py"

### PR DESCRIPTION
**Pull request by AyanT01**

### CODE REVIEW

File: hand/assets/myohand_assets.xml [hunk 1]
```diff
@@ -536,7 +536,7 @@
         <muscle name="UI_UB4" force = "76.9" class="muscle" tendon="UI_UB4_tendon" lengthrange="0.0819919 0.0969029"/>
         <muscle name="RI5"    force = "59.9" class="muscle" tendon="RI5_tendon" lengthrange="0.0425128 0.0462661"/>
         <muscle name="LU_RB5" force = "47.9" class="muscle" tendon="LU_RB5_tendon" lengthrange="0.0970142 0.100417"/>
-        <muscle name="UI_UB5" force = "47.9" class="muscle" tendon="UI_UB5_tendon" lengthrange="0.0910226 0.0976845"/>
+        <muscle name="UI_UB5" force = "48" class="muscle" tendon="UI_UB5_tendon" lengthrange="0.0910226 0.0976845"/>
```
Comment (Vittorio-Caggiano) [inline hand/assets/myohand_assets.xml:539]: can you clarify what this change is important?

Comment (Vittorio-Caggiano) [reply hand/assets/myohand_assets.xml:539]: can you please revert this change so we can merge your PR?

Also, not clear the question about how the function ? are you looking into information on initialization of the posture or how to provide muscle activations? Can you please add an issue on the MyoSuite repo to address this? 

Thanks!

Comment (AyanT01) [reply hand/assets/myohand_assets.xml]: Sure, I have reverted it!.
Yes, I'm looking for information about how to both initialize postures and how to provide muscle activations!

Thank you

**Review by Vittorio-Caggiano — APPROVED** [review]
LGTM

**Pull request merged**